docker/php/Dockerfile

```diff
@@ -0,0 +1,159 @@
+FROM php:8.3-fpm-alpine as base
+ARG UID=1000
+ARG MBIN_HOME=/var/www/mbin
+ENV MBIN_HOME=${MBIN_HOME}
+ARG MBIN_USER=mbin
+ENV MBIN_USER=${MBIN_USER}
+ARG MBIN_GROUP=www-data
+ENV MBIN_GROUP=${MBIN_GROUP}
+
+# Create user
+RUN adduser -u $UID -D -g "" $MBIN_USER $MBIN_GROUP
+# Install deps
+RUN apk update && apk add acl fcgi exiftool
+
+# Install php extensions, by docker-php-extension-installer
+COPY --from=mlocati/php-extension-installer /usr/bin/install-php-extensions /usr/local/bin/
+RUN install-php-extensions \
+      amqp \
+      apcu \
+      bcmath \
+      curl \
+      dom \
+      exif \
+      gd \
+      intl \
+      opcache \
+      pcntl \
+      pdo_pgsql \
+      pgsql \
+      redis \
+      simplexml \
+      xdebug \
+      xml
+
+# Dev will mount the entrypoint
+# Prod will embed it
+ENTRYPOINT ["/entrypoint.sh"]
+
+
+###################################################
+FROM base as dev
+# Install composer
+COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
+
+# Switch user to write as non-root into mounted local directories
+USER $MBIN_USER:$MBIN_GROUP
+
+
+######################################################
+FROM base as builder-composer
+ARG MBIN_SRC=/usr/src/mbin
+COPY --from=composer:latest /usr/bin/composer /usr/bin/composer
+
+WORKDIR $MBIN_SRC
+
+# Composer: install package
+COPY composer.* ./
+COPY symfony.lock ./
+ENV COMPOSER_ALLOW_SUPERUSER=1
+RUN composer install --prefer-dist --no-dev --no-autoloader --no-scripts --no-progress
+
+# Copy only required resources
+# TODO Use one line once COPY --parents is supported
+COPY --link assets ./assets
+COPY --link bin ./bin
+COPY --link config ./config
+COPY --link migrations ./migrations
+COPY --link public ./public
+COPY --link src ./src
+COPY --link templates ./templates
+COPY --link translations ./translations
+
+# Increase memory limit for following steps
+RUN echo "memory_limit = 512M" > /usr/local/etc/php/conf.d/memory_limit.ini
+# Must have an env file: https://github.com/symfony/symfony/issues/34660
+# Must have all env vars in it https://github.com/symfony/flex/issues/346
+COPY --link .env.example_docker .env
+
+# Dump-autoload and run post-install script
+RUN composer dump-autoload --classmap-authoritative --no-dev
+RUN composer run-script --no-dev post-install-cmd && \
+    chmod +x bin/console && sync
+
+
+######################################################
+FROM node:22.5.1-alpine3.20 as builder-nodejs
```

Comment:
Why not use `22.7.0-alpine3.20`?

Reply:
That was unchanged from the last `Dockerfile`. Dunno why github has trouble with `git mv` 🫤 

But I can upgrade it, no problem.

Reply:
Addressed in https://github.com/MbinOrg/mbin/pull/1021/commits/3bb52831faf01406df1607e728c4b2f3593c3560

Reply:
Thx as well.